server.js

```diff
@@ -42,33 +39,46 @@ app.get("/api/v2/authors", async (request, response) => {
   }
 });
 
-app.get("/api/v2/authors/:id", async (request, response) => {
-  UserHelper.findAuthorById(request.params.id).then((author) => {
-    if (author.length === 1) {
-      response.status(200).json(UserHelper.makeSecureUserResponse(author[0]));
-    } else if (author.length === 0) {
-      response.status(404).json("Author not found");
-    } else {
-      response
-        .status(500)
-        .json(
-          `Something's horribly wrong, the server has more than one author with that id`
-        );
-    }
-  });
+app.get("/api/v2/authors/:id", async (request, response, next) => {
+  try {
+    UserHelper.findAuthorById(request.params.id).then((author) => {
+      if (author.length === 1) {
+        response.header("Access-Control-Allow-Origin", "*");
+        response.status(200).json(UserHelper.makeSecureUserResponse(author[0]));
+      } else if (author.length === 0) {
+        response.header("Access-Control-Allow-Origin", "*");
+        response.status(404).json("Author not found");
+      } else {
+        response.header('Access-Control-Allow-Origin', '*')
+        response
+          .status(500)
+          .json(
+            `Something's horribly wrong, the server has more than one author with that id`
+          );
```

Comment:
Love how horribly wrong things went in this message

Reply:
You've got to have one easter egg that is probably impossible to get to.